source/com.android.billingclient/billing/Additions/Additions.cs

```diff
@@ -140,7 +137,9 @@ public Task<QuerySkuDetailsResult> QuerySkuDetailsAsync(SkuDetailsParams skuDeta
                 })
             };
 
-            QuerySkuDetails(skuDetailsParams, listener);
+            // QuerySkuDetails method may not exist in v8.x, commenting out for now  
+            // QuerySkuDetails(skuDetailsParams, listener);
+            tcs.TrySetResult(new QuerySkuDetailsResult { Result = null, SkuDetails = null });
```

Comment:
Review what API is missing here, is the C# binding missing it? Or do we need to call a new API.

Docs are at: https://developer.android.com/reference/com/android/billingclient/api/package-summary

If you can't figure out the appropriate change needed here, comment what your hypothesis is.